### PR TITLE
[ES] Translated basics/date-time chapter

### PR DESCRIPTION
**Pull request by hectorip**
I've also made a translation of the date-time chapter without noticing there was already one waiting. No hard feelings if this is closed right away, but I think this addresses the comments to the other PR.

### CODE REVIEW

**Comment by doomspork** [comment]
@erickgnavar does this need any review or is it good to merge?

File: es/lessons/basics/date-time.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+Empecemos obteniendo la hora actual:
+
+
+```elixir
+iex> Time.utc_now
+~T[19:39:31.056226]
+```
+
+Observa que también tenemos un sigil que puede ser usado para crear un estructura `Time`:
```
Comment (erickgnavar) [inline es/lessons/basics/date-time.md]: La traducción de _sigil_ sería sigilo
`un estructura` -> `una estructura`

File: es/lessons/basics/date-time.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+```elixir
+iex> ~T[19:39:31.056226]
+~T[19:39:31.056226]
+```
+
+Puedes aprender más acerca de los sigils en la [lección acerca de los sigils](../sisgils/). Es fácil acceder a partes de esta estructura:
```
Comment (erickgnavar) [inline es/lessons/basics/date-time.md]: Mismo caso para _sigils_ y también la ruta `../sisgils` esta mal

File: es/lessons/basics/date-time.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+~T[19:39:31.056226]
+iex> t.hour
+19
+iex> t.minute
+39
+iex> t.day
+** (KeyError) key :day not found in: ~T[19:39:31.056226]
+```
+
+Pero hay problemita: como te habrás dado cuenta, esta estructura sólo contine la hora del día, no hay datos acerca del día, mes o año.
```
Comment (erickgnavar) [inline es/lessons/basics/date-time.md]: `Pero hay problemita` -> `Pero un hay problemita`
`contine` -> `contiene`

File: es/lessons/basics/date-time.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+```elixir
+iex> {:ok, date} = Date.new(2020, 12,12)
+{:ok, ~D[2020-12-12]}
+iex> Date.day_of_week date
+6
+iex> Date.leap_year? date
+true
+```
+
+`day_of_week/1` calcula que día de la semana cayó una fecha dada. En este caso es sábado. `leap_year?/1` verifica si es un año bisiesto. Otras funciones pueden ser encontradas en la [documentación](https://hexdocs.pm/elixir/Date.html).
```
Comment (erickgnavar) [inline es/lessons/basics/date-time.md]: Creo que "cayó" puede ser reemplazado por "fue"

File: es/lessons/basics/date-time.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+Pero tiene tanto la hora como el la fecha actuales, así que puedes jugar añadiéndole tiempo, por ejemplo:
+
+```elixir
+iex> NaiveDateTime.add(~N[2018-10-01 00:00:14], 30)
+~N[2018-10-01 00:00:44]
+```
+
+## DateTime
+
+La segunda, coomo habrás adivinado por el título de la sección es `DateTime`.
```
Comment (erickgnavar) [inline es/lessons/basics/date-time.md]: `coomo` -> `como`

File: es/lessons/basics/date-time.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+No tiene las limitaciones observadas previamente: tiene tanto la fecha como la hora y soporta zonas horarias. Pero veamos lo que dice la documentación oficial acerca de las zonas horarias:
```
Comment (erickgnavar) [inline es/lessons/basics/date-time.md]: Se podría reemplazar los `:` por "ya que"

**Comment by erickgnavar** [comment]
Thanks for your contribution @hectorip There are some small things to be fixed.

File: es/lessons/basics/date-time.md (hunk 4 above)
Comment (hectorip) [reply es/lessons/basics/date-time.md]: De acuerdo.

File: es/lessons/basics/date-time.md (hunk 6 above)
Comment (hectorip) [reply es/lessons/basics/date-time.md]: Ok, aunque me gustan los dos puntos 😛 .

**Comment by hectorip** [comment]
@erickgnavar I've done the changes!

**Comment by hectorip** [comment]
Thanks!
  
  
  

  
  

  
  
>

**Comment by doomspork** [comment]
Thank you @hectorip!  If you're interested in staying involved, your contribution makes you eligible to join us on the all new Elixir School Slack available here: https://admissions.elixirschool.com/

**Pull request merged**